builder/builder.go

```diff
@@ -185,38 +202,36 @@ func (b *Builder) InitMix(clearver string, mixver string, all bool, upstreamurl
 	}
 	b.Mixver = mixver
 
+	// Initialize the Mix Bundles List
+	if _, err := os.Stat(filepath.Join(b.Versiondir, "mixbundles")); os.IsNotExist(err) {
+		// Add default bundles (or all)
+		defaultBundles := []string{"os-core", "os-core-update", "bootloader", "kernel-native"}
+		if err := b.AddBundles(defaultBundles, alllocal, allupstream, false); err != nil {
```

Comment:
allLocal feels easier to read

Reply:
What, you don't like three lowercase l's in a row?!